package.json

```diff
@@ -14,7 +14,9 @@
     "check": "npm run lint && npm run test:all",
     "build": "babel src --out-dir build",
     "test:watch": "mocha --compilers js:babel/register --recursive src/**/__tests__/*.js --watch",
-    "test:all": "npm run react:13 && npm test && npm run react:14 && npm test",
+    "test:describeWithDOMOnly": "mocha --compilers js:babel/register --recursive src/**/__tests__/describeWithDOM/describeWithDOMOnly-spec.js",
+    "test:describeWithDOMSkip": "mocha --compilers js:babel/register --recursive src/**/__tests__/describeWithDOM/describeWithDOMSkip-spec.js",
+    "test:all": "npm run react:13 && npm test && npm run test:describeWithDOMOnly && npm run test:describeWithDOMSkip && npm run react:14 && npm test && npm run test:describeWithDOMOnly && npm run test:describeWithDOMSkip",
```

Comment:
I suppose that's fine. I'm not 100% sure we even need to add tests for describeWithDOM, but since they're already added it's better than nothing.